src/security.c

```
@@ -1,1012 +1,616 @@
-                tag, NULL,
-                plaintext, plaintext_len,
-                associated_data, associated_data_len,
-                NULL, /* No additional nonce */
-                nonce_bytes,
-                &state) != 0)
-        {
-            return -3;
-        }
-
-        return plaintext_len;
+        uint8_t h[32];
```

Clang (14.0.6)

[warning] variable name 'h' is too short, expected at least 3 characters

(readability-identifier-length)